packaging/db/zip/linux/ignite3db

```diff
@@ -59,7 +59,20 @@ start() {
 }
 
 stop() {
```

Comment:
What about printing notification about the fact that node has successfully been stopped (or otherwise)